### PR TITLE
add Bogus to list

### PR DESCRIPTION
**Pull request by alexravenna**

### CODE REVIEW

**Review by ST-DDT — DISMISSED** [review]

**Review by xDivisionByZerox — DISMISSED** [review]
This is great, thanks. I had no idea this existed.

File: README.md [hunk 1]
```diff
@@ -77,7 +77,8 @@ Open source projects that use `@faker-js/faker` as a dependency. You can go to t
 
 ## Languages
 
-- [C++ - faker-cxx](https://github.com/cieslarmichal/faker-cxx) - C++ Faker library for generating fake (but realistic) data. 
+- [C++ - faker-cxx](https://github.com/cieslarmichal/faker-cxx) - C++ Faker library for generating fake (but realistic) data.
+- [C#/F#/VB.NET - Bogus](https://github.com/bchavez/Bogus) - A port of Faker for .NET
```
Comment (xDivisionByZerox) [inline README.md]: @ST-DDT I'm not sure about the sorting here. `+` would be equal to the ascii code 43 and `#` to 35. So techincally the entry `C#/F#/VB.NET` would come before `C++`? Like I said, I'm not sure.

Comment (ST-DDT) [reply README.md]: That is true. Having a true alphabetical sort would be easier to maintain, although it feels weird to have C# before C++.

`['C++','C#'].sort() -> [ "C#", "C++" ]`

I'm fine with both orders.

Comment (xDivisionByZerox) [reply README.md]:
```suggestion
- [C#/F#/VB.NET - Bogus](https://github.com/bchavez/Bogus) - A port of Faker for .NET
- [C++ - faker-cxx](https://github.com/cieslarmichal/faker-cxx) - C++ Faker library for generating fake (but realistic) data.
```

**Review by xDivisionByZerox — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]

**Comment by alexravenna** [comment]
> This is great, thanks. I had no idea this existed.

You're most welcome! Adding this was a suggestion from a fellow user on [daily.dev](daily.dev) 🙂.

**Pull request merged**